src/main/java/org/apache/commons/lang3/ClassUtils.java

```diff
@@ -1059,6 +1067,62 @@ public static boolean isInnerClass(final Class<?> cls) {
         return getClass(loader, className, initialize);
     }
 
+    /**
+     * Returns a list of base classes/interfaces underneath the supplied package
+     * This method only retrieves base classes/interfaces that have children that can be instantiated
+     * via a no-args constructor
```

Comment:
This line could be improved. 
At least, interfaces has no constructors.

Reply:
I have committed changes for this